brainstormforce/astra
# feat: Added Responsive Alignment control & set it to all Alignment options in customizer settings.

### Description
 - Trello task - https://trello.com/c/WlfUb5FA/1021-need-new-control-for-alignment

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
 - Check all settings if working or not listed in the checklist in the Trello task.
 - Check Backward compatibility for the same controls.

### Checklist:
